### DISCUSSION TITLE
How do I use this on a Chromebook via Codespaces

### DISCUSSION THREAD

**Discussion by AmanWithAnUnblockedPlan123940** [QuiteAFancyEmerald/InvisiProxy#400] [General]
I think we all know why im asking how to use this on a chromebook lol. But I have no clue on how to use this lol. So if it doesnt bother you itd be greatly appreciated.

**Comment by SilentisScurra** [comment]
you probably don't need it anymore but for any future people:

On the GitHub page for that proxy (make sure you are signed into GitHub using Google), look near the top right of the file list.

Click the green <> Code button.

In the dropdown menu, click the Codespaces tab.

Click the big button that says Create codespace on main (or master).

A completely new browser tab will open that looks like a dark-mode coding screen. Just wait about 1–2 minutes.

Type these comands one after another:
  npm install dotenv-flow
  cd lib/rammerhead
  npm install
  cd ../..
  npm start

Once you hit Enter on that last command, it should bypass that error completely, finish building, and give you that little popup notification in the bottom right corner (or in the Ports tab) with the green Open in Browser button.

**Reply by beanieguyvr** [reply]
it didn't work I'm still running into errors

**Reply by QuiteAFancyEmerald** [reply]
The above is outdated. Please utilize the current commands on the README for the command step.
